core/src/test/java/com/orientechnologies/orient/core/index/sbtree/local/SBTreeWALTest.java

```diff
@@ -311,13 +295,13 @@ private void restoreDataFromWAL() throws IOException {
             final long fileId = updatePageRecord.getFileId();
             final long pageIndex = updatePageRecord.getPageIndex();
 
-            OCacheEntry cacheEntry = expectedReadCache.load(fileId, pageIndex, true, expectedWriteCache, 1);
+            OCacheEntry cacheEntry = expectedReadCache.load(fileId, pageIndex, true, expectedWriteCache, 1, true);
             if (cacheEntry == null) {
```

Comment:
I suppose it should be false here. I suppose that is not a problem at all just a note.